### ISSUE TITLE
Excessive Silk Moth Spawning potentially game breaking.

### ISSUE THREAD

**Issue by MrFHering** [EnigmaticaModpacks/Enigmatica6#4954]
### Modpack Version

1.3.0

### Describe your issue.

Silk Moths Spawn from their nest infinitely and without needing a player close by so long as the chunks are loaded. In general game play this is not a problem but as I am living in the end I am running into severe lag problems with natural generated nests that spawn huge plagues of silk moths lurking just outside my render distance. I am regularly having to use commands to kill literally thousands of them.
Could they be made to despawn and or have a spawn cap  ?
 I am fearful the shear amount of moths is at some point going to make my world unplayable and I will not even be able to kill them with commands. 

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Both

**Comment by dawidmachon** [comment]
For me it is okay. When i want to settle somewhere, then you need to prepare such place. Go and clear all these chunks ;P

I just checked and on my 'end' there is no so many nests to start lagging game. For 32 chunks around my spawn place, there is maybe 10 nests ;)

**Comment by MrFHering** [comment]
Yes, I understand that but it seems pretty broken to have a mob which can just infinitely build up. One really has to find all the nests because after a day of play time that is a hell of a lot of moths from just one nest. What is more those nests are a real pain to find unless you used commands.
